apps/web/pages/api/location-autocomplete.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextApiRequest, NextApiResponse } from "next/types";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { place } = req.query;
+
```

Comment:
I think we should early return here to provide reaching out to google if this `GOOGLE_PLACES_API_KEY` is not set. 

I know it is handled and predictions will just be null but i think this a cleaner and more consistant approach

Reply:
Done